src/coreclr/vm/exinfo.cpp

```diff
@@ -300,4 +300,32 @@ void ExInfo::SetExceptionCode(const EXCEPTION_RECORD *pCER)
     DacError(E_UNEXPECTED);
 #endif // !DACCESS_COMPILE
 }
+#else // !FEATURE_EH_FUNCLETS
+
+#ifndef DACCESS_COMPILE
+ExInfo::ExInfo(Thread *pThread, CONTEXT *pCtx, REGDISPLAY *pRD, ExKind exceptionKind)
+{
+    m_pPrevExInfo = pThread->GetExceptionState()->GetCurrentExInfo();
```

Comment:
Is it possible to do this initialization in the initializer list? I'm trying to make the codebase more canonical C++ and I don't see anything that should make this hard. In fact, `m_ClauseForCatch{}` is far clearer than performing the `memset()` and avoids all sorts of UB.

Reply:
I can change it to that.